oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
 # This is a temporary fix for a Google internal issue.
 try:
-    from google.appengine.ext import ndb
```

Comment:
Anyhow, under the Google style-guide rule of "only import modules", there is only one way to go here. Will send update.